pallets/inflation-manager/src/lib.rs

```diff
@@ -180,28 +171,34 @@ pub mod pallet {
 			let inflation_config = InflationConfiguration::<T>::get();
```

Comment:
Considering the performance slightly, is it okay to move the current_year/inflation_config/inflation_parameters into the condition below?

Reply:
Good idea, i'll do these